docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md

```diff
@@ -0,0 +1,189 @@
+
+```javascript
+// setting max fee as 100 and priority fee as 2
+sendAvax("0.01", "0x856EA4B78947c3A5CD2256F85B2B147fEBDb7124", 100, 2);
+```
+**This function should not be used without a max fee per gas. As you will have to pay the estimated price, even if it is higher than your budget.**
+
+There could be the following cases -
+
+
+
+| Max Fee   | Max Priority Fee | Comment                                                                                                                                                                                                                      |
+| --------- | ---------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **undefined** | 2                | It will calculate the max fee by adding the provided priority fee with the estimated base fee. Take extra precaution here as the max fee will now be capped by `baseFee + priorityFee`, which can consume all the provided priority fees. |
+| 100       | **undefined**        | It will estimate the priority fee and use the provided max fee. If the estimated priority fee is more than the provided max fee, then it throws an error.                                                                            |
+| **undefined** | **undefined**        | It will estimate the base fee and priority fee from the network, and will add both the values to calculate the max fee per gas. Again, you have to pay whatever will be estimated.                                               |
+
+You will get the following output on the successful submission of the signed transactions. Using this URL you can view the status of your transaction on Snowtrace.
+```bash
+View transaction with nonce 25: https://testnet.snowtrace.io/tx/0xd5b92b85beaf283fbaeeefb95c9a17a6b346a05b6f9687f2d6e421aa79243b35
+```
+
+## Reissuance of stucked transaction
+
+Sometimes during high network activity, all transactions couldn't make it to the latest blocks for a long time, due to relatively lower effective tip than the other transactions in the pool. We can either re-issue the same transaction with a higher priority fee or cancel the transaction. To re-issue the stucked transaction, you can send a new one with same amount and data but higher priority fee and same nonce value as the stucked transaction. The transaction with lower effective tip will automatically be rejected (due to same nonce), and you do not need to worry about it. You can also cancel the stucked transaction, by keeping the amount to 0, with a higher priority fee and same nonce. Let's say, the above transaction with a nonce value of 25 has stucked. You can then re-issue a new transaction with same nonce, but higher priority fee this time.
+
```

Comment:
I think we should use stuck, not stucked, https://jakubmarian.com/the-word-stucked-in-english/

Reply:
lol. Will fix this.